memory/ndctl.py

```diff
@@ -274,7 +274,9 @@ def test_namespace_modes(self):
         """
         failed_modes = []
         self.enable_region()
-        region = self.get_json(short_opt='-R')[0]
+        region = self.params.get('region', default=None)
```

Comment:
Can't this be moved to setUp() ?

Reply:
yes, this particular line can be moved. but `self.get_json` in the line following cannot be used without enabling regions. So thought of keeping it together.

Reply:
Ok. I see most of the tests are using this snippet. So the question.

Reply:
This  way we can assign variable in local scope if it not using any where else and avoid to use heap memory